modules/bookAddRemove.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable */
+const titleInput = document.querySelector('.title-input');
+const authorInput = document.querySelector('.author-input');
+const addBtn = document.querySelector('.add-input');
+let bookArr = [];
+
+class Book {
+  constructor(id, title, author) {
+    this.id = id;
+    this.title = title;
+    this.author = author;
+  }
+
+  add() {
+    const book = document.createElement('div');
```

Comment:
- [ ] Kindly make use to `arrow` functions for best semantic practices 💯

Reply:
I have made the change as suggested. Thanks